src/main/java/org/broadinstitute/hellbender/tools/exome/AllelicCountCollection.java

```diff
@@ -30,24 +33,29 @@ public AllelicCountCollection() {
     }
 
     /**
-     * Constructor that reads (sequence, position, reference count, alternate count) from the specified file.
-     * @param inputFile     file to read from
+     * Constructor from from file. Checks whether the input file has just the basic columns, or full columns.
+     * @param inputFile file to read from
      */
     public AllelicCountCollection(final File inputFile) {
         Utils.nonNull(inputFile);
         Utils.regularReadableUserFile(inputFile);
+
         try (final TableReader<AllelicCount> reader = TableUtils.reader(inputFile,
```

Comment:
Instead of an inlined inner I would create a proper class called ```AllelicCountReader``` that extends ```TableReader<AllelicCount>```.